xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/vue/layouts/table/LayoutTable.vue

```diff
@@ -138,4 +144,49 @@ export default {
   padding-left: 2rem;
 }
 
+/* Table responsive mode */
+@media screen and (max-width: 767px) {
+
+  /* FOR IE11 support */
+  .layout-table thead {
+    display: flex;
+    flex-direction: row;
+  }
+  .layout-table thead tr {
+    display: flex;
+    flex-direction: column;
+  }
+  /*
+   * Align the property name with the filter
+   * 38px correspond to the height of the filters inputs
+   * This is hard coded and not scalable, but I don't have better for IE :(
```

Comment:
Add a TODO to fix then using the color theme (i.e. by moving the styles to the skin where we have a less variables with the line height).